enigma-cli/src/main/java/cuchaz/enigma/command/MappingCommandsUtil.java

```diff
@@ -59,33 +22,16 @@ public static void write(EntryTree<EntryMapping> mappings, String type, Path pat
 				throw new IllegalArgumentException("specify column names as 'tinyv2:from_namespace:to_namespace'");
 			}
 
-			new TinyV2Writer(split[1], split[2]).write(mappings, path, ProgressListener.none(), saveParameters);
-			return;
-		}
-
-		if (type.startsWith("tiny:")) {
-			String[] split = type.split(":");
-
-			if (split.length != 3) {
-				throw new IllegalArgumentException("specify column names as 'tiny:from_column:to_column'");
-			}
-
-			new TinyMappingsWriter(split[1], split[2]).write(mappings, path, ProgressListener.none(), saveParameters);
-			return;
+			return new TinyV2Writer(split[1], split[2]);
 		}
 
-		MappingFormat format = null;
+		MappingFormat format;
 		try {
-			format = MappingFormat.valueOf(type.toUpperCase());
+			format = MappingFormat.valueOf(type);
 		} catch (IllegalArgumentException ignored) {
-			// ignored!
-		}
-
-		if (format != null) {
-			format.getWriter().write(mappings, path, ProgressListener.none(), saveParameters);
-			return;
+			format = MappingFormat.valueOf(type.toLowerCase());
```

Comment:
```suggestion
			format = MappingFormat.valueOf(type.toUpperCase());
```
All of `MappingFormat`'s fields are upper case